neo/Network/P2P/Payloads/DisconnectPayload.cs

```diff
@@ -0,0 +1,36 @@
+using Neo.IO;
+using System.IO;
+
+namespace Neo.Network.P2P.Payloads
+{
+    public class DisconnectPayload : ISerializable
+    {
+        public const int MaxDataSize = 5120;
```

Comment:
1024?

Reply:
1024 is ok, as up to 200 address = 1000 bytes < 1024 bytes